src/run/formlyCustomTags.js

```diff
@@ -3,22 +3,14 @@ export default addCustomTags
 
 // @ngInject
 function addCustomTags($document) {
-  if ($document && $document.get) {
-    // IE8 check ->
-    // http://stackoverflow.com/questions/10964966/detect-ie-version-prior-to-v9-in-javascript/10965203#10965203
+  if ($document && $document.documentMode < 9) {
```

Comment:
Could you add a comment with this URL? https://msdn.microsoft.com/en-us/library/cc196988(v=vs.85).aspx